tests/Database/DatabaseEloquentBuilderTest.php

```diff
@@ -117,7 +151,7 @@ public function testFindWithManyUsingCollection()
     {
         $ids = collect([1, 2]);
         $builder = m::mock(Builder::class.'[get]', [$this->getMockQueryBuilder()]);
-        $builder->getQuery()->shouldReceive('whereIn')->once()->with('foo_table.foo', $ids);
+        $builder->getQuery()->shouldReceive('whereIn')->once()->with('foo_table.foo', [1, 2]);
```

Comment:
Why did you change this? This should stay in place.

```suggestion
        $builder->getQuery()->shouldReceive('whereIn')->once()->with('foo_table.foo', $ids);
```

Reply:
This test started to fail because now we don't pass collection any more, just an array because we made a cast

Reply:
The test is meant to proof that a collection can also pass so we need to make sure this still succeeds. Otherwise it's a breaking change.

Reply:
Well, but collection is still passed in this test, we just need to modify expectation because now into `whereIn` array is passed because now as suggested at the beginning of findMany we do:
```
$ids = $ids instanceof Arrayable ? $ids->toArray() : $ids;
```
so we pass into `whereKey` array and not collection any more and further into `whereIn` now array is passed and not collection any more.

Reply:
Right! Thanks for point that out :)